docs/user/alerting/stack-alerts/index-threshold.asciidoc

```diff
@@ -48,71 +48,56 @@ When the index threshold alert condition is met, the following variables are ava
 `context.value`:: The value for the alert that met the threshold condition.
 `context.conditions`:: A description of the threshold condition. Example: `count greater than 4`
 
+<<defining-alerts-actions-details, Add an action>> to the alert to use these variables.
+
 [float]
 ==== Example
 
 In this section, you will use the {kib} <<add-sample-data, weblog sample dataset>> to setup and tune the conditions on an index threshold alert. For this example, we want to detect when any of our top three sites have served more than 420,000 bytes over a 24 hour period.
 
-From the <<alert-management, alert management UI>>, create a new alert, and fill in the <<defining-alerts-general-details, general alert details>>. This alert will be checked every 4 hours, and will not execute actions more than once per day. Choose the index threshold alert type. 
-
+. From the <<alert-management, Alerts and Actions>>, create a new alert and fill in the <<defining-alerts-general-details, general alert details>>. This alert will be checked every 4 hours, and will only execute actions when the alert status changes. Choose the index threshold alert type. 
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-select.png[Choosing an index threshold alert type]
 
-Click on each clause to open a control that helps you set the value:
-
-[float]
-==== Index clause
```

Comment:
I suggest making each of these a step.

4. Click **Index**, and set **Indices to query** to **kibana_sample_data_logs.**
5.  Set the **Time field** to **@timestamp.**
6. To detect the number of bytes served during the time window, click **When** and select sum as the aggregation, and bytes as the field to aggregate.

and so on.  If you end up with too many steps, you can break it into sections.